test/helper.rb

```diff
@@ -9,11 +9,30 @@
 
 # Test setup
 MIGRATIONS_ROOT = 'test/migrations'
-Mongoid.configure.connect_to('mongoid_test')
+
+Mongoid.configure.load!("#{__dir__}/mongoid.yml", 'test')
 require 'models/survey_schema'
 
 module TestMongoidRailsMigrations
   class Application < Rails::Application; end
 end
 
 TestMongoidRailsMigrations::Application.load_tasks
+
+# Hide task output
+class Mongoid::Migration
+  def self.puts _
+  end
+end
+
+def invoke(task)
+  Rake.application.tasks.each(&:reenable)
+  Rake::Task[task].invoke
+end
+
+def with_env(options)
+  options.keys.each { |key| ENV[key] = options[key] }
```

Comment:
```suggestion
  options.each { |k, v| ENV[k] = v }
```

Really minor but cleaner.